hazelcast/src/main/java/com/hazelcast/config/Config.java

```diff
@@ -200,57 +220,150 @@ public String getProperty(String name) {
      *
      * @param name  property name
      * @param value value of the property
-     * @return configured {@link Config} for chaining
+     * @return this config instance
      */
     public Config setProperty(String name, String value) {
         properties.put(name, value);
         return this;
     }
 
+    /**
+     * Returns the member attribute configuration. Unlike the config
+     * properties, member attributes can be serialised and exchanged with
+     * other members.
+     *
```

Comment:
I'm totally confused from this. "Property" means one set by `setProperty`? That's String, it is serializable. I don't know.
It should also refer to documentation of possible properties.

Reply:
Yes, I was trying to say that the member attributes *will* be serialised and exchanged with other members while the `Config.getProperties()` are strictly local. I'll clarify this.